src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -782,91 +790,65 @@ def apply():
+        4. If ``func`` is a series-to-series function that can also be used as a scalar-to-scalar function
+        (e.g., ``np.sqrt``, ``lambda x: x+1``), using ``df.applymap()`` to apply the function
+        element-wise may give better performance.
 
         5. When ``func`` can return a series with different indices, e.g.,
-        ``lambda x: pd.Series([1, 2], index=["a", "b"] if x.sum() > 2 else ["b", "c"]).``,
+        ``lambda x: pd.Series([1, 2], index=["a", "b"] if x.sum() > 2 else ["b", "c"])``,
         the values with the same label will be merged together.
 
-        6. The index values of returned series from ``func`` must be json-serializable. For example,
-        ``lambda x: pd.Series([1], index=[bytes(1)])`` will raise a SQL execption.
+        6. The index values of returned series from ``func`` must be JSON-serializable. For example,
+        ``lambda x: pd.Series([1], index=[bytes(1)])`` will raise a SQL execption because python ``bytes``
+        objects are not JSON-serializable.
 
         7. When ``func`` uses any first-party modules or third-party packages inside the function,
         you need to add these dependencies via ``session.add_import()`` and ``session.add_packages()``.
         Alternatively. specify third-party packages with the @udf decorator. When using the @udf decorator,
         annotations using PandasSeriesType or PandasDataFrameType are not supported.
 
-        8. Snowpark pandas module is currently not supported inside ``func``. If you need to call
-        general pandas API (e.g., ``pd.Timestamp``) inside ``func``, the workaround is to use
-        the actual pandas module.
+        8. The Snowpark pandas module cannot currently be referenced inside the definition of
+        ``func``. If you need to call a general pandas API like ``pd.Timestamp`` inside ``func``,
+        please use the original ``pandas`` module (with ``import pandas``) as a workaround.
 
         Examples
         --------
-        >>> import snowflake.snowpark.modin.pandas as pd
         >>> df = pd.DataFrame([[2, 0], [3, 7], [4, 9]], columns=['A', 'B'])
         >>> df
            A  B
         0  2  0
         1  3  7
         2  4  9
 
-        Using a numpy universal function (in this case the same as
-        ``np.sqrt(df)``):
+        Using a reducing function on ``axis=1``:
 
-        >>> df.apply(np.sqrt)  # doctest: +SKIP
-                  A         B
-        0  1.414062  0.000000
-        1  1.732422  2.646484
-        2  2.000000  3.000000
-
-
-        Using a reducing function on either axis
-
-        >>> df.apply(np.sum, axis=0)  # doctest: +SKIP
-        A     9
-        B    16
-        dtype: int8
-
-        >>> df.apply(np.sum, axis=1)  # doctest: +SKIP
+        >>> df.apply(np.sum, axis=1)
         0     2
         1    10
         2    13
         dtype: int64
 
-        Returning a list-like will result in a Series
+        Returning a list-like result will result in a Series:
```

Comment:
I'd leave this as is because (1) it's the original wording in the pandas docs, and (2) ` ... result will result ...` doesn't sound very clear.

Reply:
I'll change it to "list-like object", I tend to think of "list-like" as an adjective, and it feels like a bit of a [garden-path sentence](https://en.wikipedia.org/wiki/Garden-path_sentence) in the original wording.